tests/foreman/ui/test_user.py

```diff
@@ -615,7 +615,7 @@ def test_positive_update_language(self):
 
         @expectedresults: User is updated successfully
         """
-        locale = random.choice(LANGUAGES)
+        locale = random.choice(LANGUAGES.values())
```

Comment:
@renzon, this is causing the test to fail, since it now filters the locales by the abbreviations, while the actual locales in the dropdown are in a full-text form.